indexing-service/src/test/java/io/druid/indexing/overlord/autoscaling/EC2AutoScalerTest.java

```diff
@@ -72,6 +83,7 @@
   public void setUp() throws Exception
   {
     amazonEC2Client = EasyMock.createMock(AmazonEC2Client.class);
+    mockAmazonEC2Client = new EC2MockClient();
```

Comment:
can you use EasyMock instead of creating a new class ? 
It doesn't seem to do anything custom else except get/set region.